### DISCUSSION TITLE
homebridge login and password

### DISCUSSION THREAD

**Discussion by domoticauser** [tteck/Proxmox#1739] [Q&A] [answered]
Hi,
What is the username and password to access with ssh?

Regards
Mark

**Comment by tteck** [comment, marked as answer]
Was SSH enabled when you were setting it up using the "Advanced" settings? If you used "Default" settings, then there's no straightforward SSH access available.

https://github.com/tteck/Proxmox/blob/f5b629de5175e1669d2376ecb4357410aeaac6b4/ct/homebridge.sh#L48
